timesketch/lib/analyzers/feature_extraction.py

```diff
@@ -229,7 +265,43 @@ def extract_feature(self, name, config):
                 continue
 
             event_counter += 1
-            event.add_attributes({store_as: result[0]})
+            #verify if store_as exist
+            store_type = event.source.get(store_as)
+            if store_type and not overwrite_store_as:
+                continue
+            if isinstance(store_type, six.text_type):
+                store_type_list = False
+            elif isinstance(store_type, (list, tuple)):
+                store_type_list = True
+            if store_type_list:
+                if keep_multimatch:
+                    if store_type and overwrite_and_merge_store_as:
+                        #don't add duplicate value
+                        resultList = list(set(store_type) | set(result))
+                        event.add_attributes({store_as: resultList})
+                    else:
+                        event.add_attributes({store_as: result})
+                else:
+                    if overwrite_and_merge_store_as and store_type and result[0] in store_type:
+                        continue
+                    event.add_attributes({store_as: [result[0]]})
+            else:
+                if keep_multimatch:
+                    #TODO: add option to choice join char
```

Comment:
keep a space between # and the TODO

also can you further elaborate on what the todo is about?

Reply:
I removed, it was for change join char by option:
```
event.add_attributes({store_as: ','.join(result)})
```